### PR TITLE
DOCSUP-10514: documented new setting prefer_global

### PR DESCRIPTION
**Pull request by atereh**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #23434

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -20,6 +20,29 @@ Possible values:
...
+
+**Usage**
+
+Although `SET distributed_product_mode=global;` can change the queries behavior for the distributed tables, it's not suitable for local tables or tables from external resources. Here is when the `prefer_global_in_and_join` setting comes into play.
+
+For example, we have query serving nodes that contain local tables, which are not suitable for distribution. We need to scatter their data on the fly during distributed processing with the `GLOBAL` keyword — `GLOBAL IN`/`GLOBAL JOIN`.
+
+Another use case of `prefer_global_in_and_join` is accessing tables created by external engines. This setting helps to reduce the number of calls to external sources while joining such tables: only one call per query.
+
+**Check Also**
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:42]:
```suggestion
**See Also**
```

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -20,6 +20,29 @@ Possible values:
...
 -   `allow` — Allows the use of these types of subqueries.
 
+## prefer_global_in_and_join {#prefer-global-in-and-join}
+
+Enables the replacement of `IN`/`JOIN` operators with `GLOBAL IN`/`GLOBAL JOIN`.
+
+Possible values:
+
+-   0 — Disabled.
+-   1 — Enabled.
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:30]:
```suggestion
-   0 — `IN`/`JOIN` operators are not replaced with `GLOBAL IN`/`GLOBAL JOIN`.
-   1 — `IN`/`JOIN` operators are replaced with `GLOBAL IN`/`GLOBAL JOIN`.
```

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -20,6 +20,29 @@ Possible values:
...
+
+Default value: `0`.
+
+**Usage**
+
+Although `SET distributed_product_mode=global;` can change the queries behavior for the distributed tables, it's not suitable for local tables or tables from external resources. Here is when the `prefer_global_in_and_join` setting comes into play.
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:36]:
```suggestion
Although `SET distributed_product_mode=global` can change the queries behavior for the distributed tables, it's not suitable for local tables or tables from external resources. Here is when the `prefer_global_in_and_join` setting comes into play.
```

File: docs/en/operations/settings/settings.md [hunk 4]
```diff
@@ -20,6 +20,29 @@ Possible values:
...
+
+**Usage**
+
+Although `SET distributed_product_mode=global` can change the queries behavior for the distributed tables, it's not suitable for local tables or tables from external resources. Here is when the `prefer_global_in_and_join` setting comes into play.
+
+For example, we have query serving nodes that contain local tables, which are not suitable for distribution. We need to scatter their data on the fly during distributed processing with the `GLOBAL` keyword — `GLOBAL IN`/`GLOBAL JOIN`.
+
+Another use case of `prefer_global_in_and_join` is accessing tables created by external engines. This setting helps to reduce the number of calls to external sources while joining such tables: only one call per query.
+
+**See Also**
```
Comment (mathalex) [inline docs/en/operations/settings/settings.md:42]:
```suggestion
**See also:**
```

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -25,6 +25,30 @@ ClickHouse применяет настройку в тех случаях, ко
...
+**Использование**
+
+Настройка `SET distributed_product_mode=global` меняет поведение запросов для распределенных таблиц, но она не подходит для локальных таблиц или таблиц из внешних источников. В этих случаях удобно использовать настройку `prefer_global_in_and_join`.
+
+Например, если нужно объединить все данные из локальных таблиц, которые находятся на разных узлах — для распределенной обработки необходим `GLOBAL JOIN`.
+
+Другой вариант использования настройки `prefer_global_in_and_join` — регулирование обращений к таблицам из внешних источников.
+Эта настройка помогает уменьшить количество обращений к внешним ресурсам при объединении внешних таблиц: только один вызов на весь распределенный запрос.
+
+**Смотрите также**
```
Comment (mathalex) [inline docs/ru/operations/settings/settings.md:48]:
```suggestion
**См. также:**
```

**Review by mathalex — COMMENTED** [review]
For 'also' should be used the lowercase letter. Other fixes are optional. Eventually, both files contain these words in different styles (that should be fixed in the future for uniformity).

**Comment by atereh** [comment]
@mathalex thank you for pointing the issue with inconsistency! I will discuss the solution with our documentation team

**Review by mathalex — APPROVED** [review]

**Pull request merged**